### ISSUE TITLE
Make Search as I Type work with pdf's as well

### ISSUE THREAD

**Issue by FatOrangutan** [Quicksaver/FindBar-Tweak#220]
Would it be possible to make the 'search as I start typing' feature work with pdf's opened with FF's pdf.js viewer as well? I realize that this is how FF handles this feature by default / without FindBar Tweak as well, but it seems counter-intuitive to have the feature work on all pages except pdf's.
Thanks.
Edit: I just noticed that there is a 'Enable Quick Find in PDF files' in the roadmap section on your website. It seems that it is the same as my request. Hence, I'm closing this issue.

**Comment by Quicksaver** [comment]
Yep, it's already planned as you could see from the roadmap, but you can leave this issue open since there isn't one yet for this enhancement. It'll help others know this more easily as well.
